tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -75,6 +75,7 @@ def hash_to_patterns(hash)
     if /^\[.*\]$/.match(value)
       value.gsub!(/[\[\]]/) { |s| '\\' + "#{s}" }.gsub!(/\"/) { |_s| '\'' }
     end
+    value.gsub!(/[\(\)]/) { |s| '\\' + "#{s}" } if /\(.*\)/.match(value)
```

Comment:
My concern is that this change will break other tests that use this method.  Probably need to to run a full beaker test to make sure.